src/main/scala/io/shiftleft/js2cpg/io/FileUtils.scala

```diff
@@ -102,6 +104,16 @@ object FileUtils {
             }
             result
           }
+
+          override def visitFileFailed(file: Path, exc: IOException): FileVisitResult = {
+            exc match {
+              case loop: FileSystemLoopException =>
+                logger.debug(s"Cyclic symbolic link detected for file '$file'", loop)
```

Comment:
1) Display the action we take, i.e. we're simply ignoring this file
2) This is just an info (or rather, debug) message, so logging an exception stacktrace may lead users to think this is some severe error

```suggestion
                logger.debug(s"Cyclic symbolic link detected for file '$file' - ignoring")
```